lib/stream_data.ex

```diff
@@ -2042,6 +2047,40 @@ defmodule StreamData do
     ])
   end
 
+  @doc """
+  Generates chardata.
+
+  Chardata are values of the `t:IO.chardata/0` type.
+
+  ## Examples
+
+      Enum.take(StreamData.chardata(), 3)
+      #=> ["", [""], [12174]]
+
+  ## Shrinking
+
+  Shrinks towards less nested chardata and ultimately towards smaller binaries.
+  """
+  @spec chardata() :: t(IO.chardata())
+  def chardata() do
+    codepoint = @utf8_chars |> Enum.map(&{Enum.count(&1), integer(&1)}) |> frequency()
```

Comment:
I guess there could be a case for a `codepoint/0` generator, let me know your thoughts

Reply:
Yep, we can add it in another PR if that's ok with you!

Reply:
Sure, I'm on it!

Reply:
Reconsidering it, should it rather be `codepoint/1` and accept `:utf8 | :printable | :ascii | :alphanumeric`? 🤔

Reply:
Yes, that sounds good 👍